docs/source/en/using-diffusers/tips.md

```diff
@@ -0,0 +1,160 @@
+    <figcaption class="mt-2 text-center text-sm text-gray-500">step 0</figcaption>
+  </div>
+  <div>
+    <img class="rounded-xl" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/tips_step_19.png"/>
+    <figcaption class="mt-2 text-center text-sm text-gray-500">step 19
+    </figcaption>
+  </div>
+  <div>
+    <img class="rounded-xl" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/tips_step_29.png"/>
+    <figcaption class="mt-2 text-center text-sm text-gray-500">step 29</figcaption>
+  </div>
+  <div>
+    <img class="rounded-xl" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/tips_step_39.png"/>
+    <figcaption class="mt-2 text-center text-sm text-gray-500">step 39</figcaption>
+  </div>
+  <div>
+    <img class="rounded-xl" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/tips_step_49.png"/>
+    <figcaption class="mt-2 text-center text-sm text-gray-500">step 49</figcaption>
+  </div>
+</div>
+
+## High quality anime images
+
+> [!TIP]
+> This tip was contributed by [asomoza](https://github.com/asomoza).
+
+Generating high-quality anime images is a very popular application of diffusion models. To achieve this in Diffusers:
+
+1. Choose a good anime model like [Counterfeit](https://hf.co/gsdf/Counterfeit-V3.0) and pair it with negative prompt embeddings such as [EasyNegative](https://huggingface.co/embed/EasyNegative) to further improve the quality of the generated images.
+
+```py
+from diffusers import StableDiffusionPipeline, DPMSolverMultistepScheduler
+import torch
+
+pipeline = StableDiffusionPipeline.from_single_file(
+    "https://huggingface.co/gsdf/Counterfeit-V3.0/blob/main/Counterfeit-V3.0_fix_fp16.safetensors",
+    torch_dtype=torch.float16,
+)
+pipeline.load_textual_inversion(
+    "embed/EasyNegative",
+    weight_name="EasyNegative.safetensors",
+    token="EasyNegative"
+)
+```
+
+2. Download the weights (typically a LoRA adapter) of a specific style to apply to the image and use the [`load_lora_weights`] method to add it to the pipeline. This example uses the [Dungeon Meshi Marcille Character Lora](https://civitai.com/models/106199/dungeon-meshi-marcille-character-lora).
+
+```py
+!wget https://civitai.com/api/download/models/114049 -O marcille.safetensors
+pipeline.load_lora_weights('.', weight_name="marcille.safetensors")
+```
+
+3. Load a scheduler and set `use_karras_sigmas=True` to use the DPM++ 2M Karras scheduler (take a look at this [scheduler table](../api/schedulers/overview.) to find the A1111 equivalent scheduler in Diffusers).
+
+```py
+pipeline.scheduler = DPMSolverMultistepScheduler.from_config(pipeline.scheduler.config)
+pipeline.scheduler.config.use_karras_sigmas=True
+pipeline.to("cuda")
+```
```

Comment:
```suggestion
```py
pipeline.scheduler = DPMSolverMultistepScheduler.from_config(pipeline.scheduler.config)
pipeline.scheduler.config.use_karras_sigmas = True
pipeline.to("cuda")
```
```